Jenkinsfile

```diff
@@ -22,6 +31,7 @@ node('rhel7'){
 		def serverUrl = "https://repo.eclipse.org/content/repositories/lemminx-releases/org/eclipse/lemminx/org.eclipse.lemminx/${serverVersion}/org.eclipse.lemminx-${serverVersion}-uber.jar"
 		sh "curl -Lo org.eclipse.lemminx-${serverVersion}-uber.jar ${serverUrl}"
 		files = findFiles(glob: 'org.eclipse.lemminx*-uber.jar')
+		buildLemMinXBinary()
```

Comment:
need to download the NativeImage.jenkins file at this point since vscode-xml has not yet been cloned